db/sql_changeset_schema

```diff
@@ -38,7 +38,12 @@ ALTER TABLE user_accounts ADD CONSTRAINT email_uniq UNIQUE (email);
 CREATE INDEX user_account_devices_userid_idx on user_account_devices(userid);
 CREATE INDEX user_files_userid_idx on user_files(userid);
 CREATE INDEX user_files_deviceid_idx on user_files(deviceid);
-CREATE INDEX user_files_shared_code_idx on user_files(shared_code);
+
+-- share file
+CREATE TABLE share_files(code varchar NOT NULL primary key, name text NOT NULL, type text NOT NULL, userid integer NOT NULL, info jsonb);

```

Comment:
1. `varchar` could be enough for `type`
2. it seems to be a good idea to add `expire` as `timestamp`